homeassistant/components/prusalink/camera.py

```diff
@@ -35,7 +35,12 @@ def __init__(self, coordinator: JobUpdateCoordinator) -> None:
     @property
     def available(self) -> bool:
         """Get if camera is available."""
-        return super().available and self.coordinator.data.get("job") is not None
+        return (
+            super().available
+            and self.coordinator.data.get("file") is not None
+            and self.coordinator.data.get("file").get("refs").get("thumbnail")
+            is not None
```

Comment:
The camera entity has a `JobUpdateCoordinator`, which returns a `JobInfo`, which has no `file` attribute declared. Are you sure the code is fine? If yes, please also update the typed dict

```suggestion
            and (file := self.coordinator.data.get("file"))
            and file.get("refs").get("thumbnail")
```

I assume an empty string is also invalid for `thumbnail` as it is used below

Reply:
I've applied the suggestion. Sadly, this doesn't resolve the issue that is already present with the current version #104887. The printer will always report that there is a thumbnail available, but I'll fix that in an upcoming PR based on my other PR: https://github.com/home-assistant-libs/pyprusalink/pull/72 + Core PR in the future.